.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/Users/bdizzle/Desktop/codebases/4-mod/fee-dreamhome/src/components/Sourvey/Sourvey.tsx":"1"},{"size":934,"mtime":1609981698141,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":1,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"13szocz","/Users/bdizzle/Desktop/codebases/4-mod/fee-dreamhome/src/components/Sourvey/Sourvey.tsx",["6"],{"ruleId":null,"fatal":true,"severity":2,"message":"7","line":1,"column":0},"Parsing error: Merge conflict marker encountered."]
```

Comment:
I don't like how this file is changing every time we switch from one branch to another.

Reply:
I'm not sure what I did, but it's not an issue for me anymore and not injecting itself into every commit. We can try to figure it out when we meet.